### PR TITLE
Docs - inclusive terminology update

### PR DESCRIPTION
**Pull request by dyozie**
Changing "disable" and variations to "deactivate".  Also changes occurrences of enable with activate in close proximity to "disable".

### CODE REVIEW

File: gpdb-doc/markdown/admin_guide/load/topics/g-defining-a-command-based-writable-external-web-table.html.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ The following Greenplum Database variables are available for use in OS commands
 |$GP\_USER|The database user running the external table statement.|
 |$GP\_XID|The transaction ID of the external table statement.|
 
--   **[Disabling EXECUTE for Web or Writable External Tables](../../load/topics/g-disabling-execute-for-web-or-writable-external-tables.html)**  
+-   **[Disabling EXECUTE for Web or Writable External Tables](../../load/topics/g-deactivating-execute-for-web-or-writable-external-tables.html)**
```
Comment (lisakowen) [inline gpdb-doc/markdown/admin_guide/load/topics/g-defining-a-command-based-writable-external-web-table.html.md:40]: i don't think you changed the name of the file, so probably want to retain the original file name.

File: gpdb-doc/markdown/admin_guide/query/topics/query-piv-opt-enable.html.md [hunk 2]
```diff
@@ -2,15 +2,15 @@
 title: Enabling and Disabling GPORCA 
 ---
 
-By default, Greenplum Database uses GPORCA instead of the Postgres Planner. Server configuration parameters enable or disable GPORCA.
+By default, Greenplum Database uses GPORCA instead of the Postgres Planner. Server configuration parameters activate or disactivate GPORCA.
```
Comment (lisakowen) [inline gpdb-doc/markdown/admin_guide/query/topics/query-piv-opt-enable.html.md:5]: did you mean to use "disactivate" here?

File: gpdb-doc/markdown/admin_guide/load/topics/g-defining-a-command-based-writable-external-web-table.html.md (hunk 1 above)
Comment (dyozie) [reply gpdb-doc/markdown/admin_guide/load/topics/g-defining-a-command-based-writable-external-web-table.html.md:40]: Thanks for that catch!  Also missed re-wording the actual link text here.

File: gpdb-doc/markdown/admin_guide/query/topics/query-piv-opt-enable.html.md (hunk 2 above)
Comment (dyozie) [reply gpdb-doc/markdown/admin_guide/query/topics/query-piv-opt-enable.html.md:5]: Nope - clumsy typing!

File: gpdb-doc/markdown/ref_guide/modules/diskquota.html.md [hunk 3]
```diff
@@ -181,7 +181,7 @@ You can perform the following tasks with the `diskquota` module:
 -   [Set a Tablespace Disk Quota for a Schema or Role](#tablespace_quota)
 -   [Set a Per-Segment Tablespace Disk Quota](#per_seg_tblsp_quota)
 -   [Display Disk Quotas and Disk Usage](#quotas_usage)
--   [Temporarily Disable Disk Quota Monitoring](#temp_disable)
+-   [Temporarily Deactivate Disk Quota Monitoring](#temp_deactivate)
```
Comment (lisakowen) [inline gpdb-doc/markdown/ref_guide/modules/diskquota.html.md:184]: could changing the tag cause any issues for existing xrefs?

**Review by lisakowen — APPROVED** [review]
thanks for this work!

looks good.  there are a few `disactivate`s sprinkled in the files, not sure if this was intentional.

File: gpdb-doc/markdown/ref_guide/modules/diskquota.html.md (hunk 3 above)
Comment (dyozie) [reply gpdb-doc/markdown/ref_guide/modules/diskquota.html.md:184]: This file has the only use of that anchor, so should be fine.

**Pull request merged**